operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md

```diff
@@ -90,47 +122,55 @@ A DNS SRV record is structured as follows:
 - Name: The domain name for which this record is valid.
 - Other elements were omitted for clarity and brevity.
 
+```text
+<service>.<protocol>.<domain name>
+```
+
 For example, DNS SRV record might look like this:
 
 ```text
-_sbps-hsn._tcp.my-system.my-site-domain.net
+_sbps-hsn._tcp.my-system.my-site-domain
 ```
 
 The following is an explanation of the values used in this example:
 
 - `_sbps-hsn` is the symbolic name of the service.
+  - In this example, the High Speed Network (hsn) is being used.
 - `_tcp` is the transport protocol.
-- `my-system.my-site-domain.net` is the domain name for which the record is valid. The domain name includes the system name `my-system` and the site domain name `my-site-domain.net`.
+- `my-system.my-site-domain` is the domain name for which the record is valid.
+  - The domain name includes the system name `my-system` and the site domain name `my-site-domain`.
+
+To use the Node Management Network (NMN) for content projection, the service is set to `_sbps-nmn`.
+  - `"rootfs_provider_passthrough"`: Set to `"sbps:v1:iqn.2023-06.csm.iscsi:_sbps-nmn._tcp.my-system.my-site-domain:300"`
+  
+To use the High Speed Network (HSN) for content projection, the service is set to `_sbps-hsn`.
```

Comment:
I was just thinking that it will be good if we can add additional details on HSN being default Network for customers w.r.t iSCSI SBPS and state high-level differences between usage of HSN vs NMN here. Any thoughts or out of scope for this document?

Reply:
@ravikanth-nalla-hpe , let's discuss this further. I recently had a discussion with Chris Spiller and others. He was fine with using the NMN for booting. Other sites like Frontier, which are massive, have successfully used the NMN for booting. I was unaware that the HSN was the default network for iSCSI. I think we should figure this out. This probably requires a meeting. I'll set one up.

Reply:
We're handling this in e-mail.